backend/src/v5/models/revisions.js

```diff
@@ -67,4 +67,13 @@ Revisions.updateRevisionStatus = async (teamspace, model, revision, status) => {
 	}
 };
 
+Revisions.isValidTag = async (teamspace, model, tag) => {
```

Comment:
Naming of the function is a bit weird, what we're testing here is if tag is already in use. so maybe it's better to call it `tagInUse` or something